views/group/show.pug

```diff
@@ -93,8 +93,8 @@ block content
     table.table.is-fullwidth.is-hoverable
       thead
         .columns
-          .column.menu-label Résztvevők
-          if !joined && !group.doNotDisturb
+          .column.menu-label Résztvevők !{group.users.length} / !{group.maxAttendees || 100}
```

Comment:
It's preferable to use the `#{variable}` syntax, as it escapes any potential text. Also, please remove the `|| 100` expressions here and in line 97 as they're not needed with the db-level default.